src/plugins/intel_cpu/tests/functional/single_layer_tests/matmul.cpp

```diff
@@ -325,6 +345,14 @@ const std::vector<ShapeRelatedParams> IS2D_nightly = {
     },
 };
 
+std::vector<fusingSpecificParams> filterFusingParams(const std::vector<fusingSpecificParams>& orig) {
+#ifndef OV_CPU_WITH_MLAS
```

Comment:
It disables fusing for all the matmul tests when compiled with MLAS, doesn't it? May be it does make sense to move MLAS specific instances to a separate folder?

Reply:
I agree that we should setup specific tests for MLAS. However current MLAS enabled-FC could intercept the FC tests for oneDNN. We don't have a switch to turn off MLAS in original FC tests. Therefore, I need to filter out those tests designed for oneDNN.

Reply:
Agree to leave it as it is as an intermediate solution. But, when we introduce heuristics to select oneDNN/MLAS at runtime (which seems to be inevitable), we will have to turn to the separate test instances solution anyway.